### DISCUSSION TITLE
Is Orkes conductor community paid?

### DISCUSSION THREAD

**Discussion by lyoumi** [orkes-io/orkes-conductor-community#55] [Q&A]
I currently doing research for an all available orchestration tools over the market. AFAIS Orkes provides variety of different SaaS solutions that can be used if company decide to not manage it. On the other hand Orkes Conductor Community is based on Apache 2.0 (https://github.com/orkes-io/licenses/blob/main/community/LICENSE.txt) and can be used for commercial purpose but company must manage it by itself?

**Comment by RussellTaylor83** [comment]
It is free to use. It hasn't seen a new release in 11 months now, so I don't believe it's being actively developed and probably should be avoided. I would look to conductor-oss instead.
